lib/winapi/processthreadsapi.h

```diff
@@ -1,16 +1,31 @@
 #ifndef __PROCESSTHREADSAPI_H__
 #define __PROCESSTHREADSAPI_H__
 
+#include <winbase.h>
 #include <windef.h>
 
 #ifdef __cplusplus
-extern "C" {
+extern "C"
+{
 #endif
 
+typedef DWORD (__stdcall *LPTHREAD_START_ROUTINE) (LPVOID lpThreadParameter);
+// The STACK_SIZE_PARAM_IS_A_RESERVATION flag has no effect, stacks are always committed memory in the Xbox
```

Comment:
Comment should be with the constant; [also affects `_beginthread`](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/beginthread-beginthreadex?view=vs-2019) (which is where I originally noticed the lack of support)

Reply:
Done.